Header header                        # header for time/frame information
string ns                            # Namespace to place this object in... used in conjunction with id to create a unique name for the object
int32 id                             # object ID

float64 getVoxelGridFromPointCloud
float64 updateFromOFlow
float64 getMeasurementModel
float64 prediction
float64 measurementBasedUpdate
float64 segment
float64 updateSpeedFromObstacles
float64 initialization
float64 totalCompute
float64 totalVisualization